src/main/java/openperipheral/integration/forestry/AdapterBeeHousing.java

```diff
@@ -61,24 +61,26 @@ public IIndividual getQueen(IBeeHousing beeHousing) {
 
 	@Asynchronous
 	@ScriptCallable(returnTypes = ReturnType.TABLE, description = "Get the full breeding list thingy. Experimental!")
-	public List<Map<String, Object>> getBeeBreedingData(IBeeHousing housing) {
+	public List<Map<String, Object>> getBeeBreedingData(IBeeHousing housing, @Optionals @Arg(name = "showSecrets", type = ArgType.BOOLEAN) Boolean showSecrets) {
```

Comment:
1. Forestry bee breeding is niche thing and only determined individuals pursue that. Those can ask themselves if they want secrets or not, because they will see all secret species in NEI for sure.
2. ComputerCraft programming is other niche thing and persons who use it most probably already read all the wiki's and on discovery of secret bees they would read about them as well.
3. Current behavior of current release version of OpenPeripheralIntegration is to show all species and all mutations disregarding isSecret() on either of them.

Conclusion – this commit pushes the balance a little to conspiracy side, just as much as it need be. Also OpenPeripheralIntegration have no out-of-the-game documentation system.

You are just being over-protective here. If you want to keep secret a secret, go and report every YouTuber who showed your secret species in their videos.